pkg/ery/di/daemon_providers.go

```diff
@@ -0,0 +1,17 @@
+package di
+
+import (
+	"github.com/google/go-cloud/wire"
+	"github.com/srvc/ery/pkg/app/daemon"
+	"github.com/srvc/ery/pkg/ery"
+)
+
+func ProvideDaemonFactory(cfg *ery.Config) daemon.Factory {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideDaemonFactory should have comment or be unexported